src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -1721,9 +1722,16 @@ private List<CompletionResult> GetResultForString(CompletionContext completionCo
 
                 foreach (var keyword in matchedResults)
                 {
-                    string usageString = Microsoft.PowerShell.DesiredStateConfiguration.Internal.CrossPlatform.DscClassCache.NewApiIsUsed
-                        ? Microsoft.PowerShell.DesiredStateConfiguration.Internal.CrossPlatform.DscClassCache.GetDSCResourceUsageString(keyword)
-                        : Microsoft.PowerShell.DesiredStateConfiguration.Internal.DscClassCache.GetDSCResourceUsageString(keyword);
+                    string usageString = string.Empty;
+                    ICrossPlatformDsc dscSubsystem = SubsystemManager.GetSubsystem<ICrossPlatformDsc>();
+                    if ((dscSubsystem != null) && (dscSubsystem.NewApiIsUsed))
```

Comment:
```suggestion
                    if (dscSubsystem?.NewApiIsUsed)
```

Reply:
I think the correct syntax is `if (dscSubsystem?.NewApiIsUsed ?? false)`
Changed to that.